docs/user/security/fips-140-2.asciidoc

```diff
@@ -0,0 +1,62 @@
+[role="xpack"]
+[[xpack-security-fips-140-2]]
+=== FIPS 140-2
```

Comment:
Heya @georgewallace ! Would you or someone on you team be able to take a look at this documentation?

Reply:
Heya @elastic/kibana-docs Would someone mind taking a look?

Reply:
To make this page appear in the output, you'll need to add `include::fips-140-2.asciidoc[]` at the end of the `docs/user/security/index.asciidoc` file (or in some other navigation file, depending on where it ought to appear in the navigation tree).

Reply:
I've added the page to the navigation in c37004ac14ef151b52f138a1fe649f93176fdee1.  I think it could honestly go in either of these sections: https://www.elastic.co/guide/en/kibana/current/using-kibana-with-security.html or https://www.elastic.co/guide/en/kibana/current/xpack-security.html.  I've added it to the latter for now but can easily move if the former makes more sense for this content.